src/content/it/2020/seo.md

```diff
@@ -0,0 +1,1217 @@
+
+Tutte queste cose hanno probabilmente contribuito al tasso di adozione più elevato anno dopo anno.
+
+{{ figure_markup(
+  image="seo-percentage-of-https.png",
+  caption="Percentuale di pagine pubblicate con HTTPS.",
+  description="Grafico a barre che mostra la percentuale di pagine che utilizzano il protocollo HTTPS per dispositivo. Abbiamo rilevato che il 77.44% delle pagine desktop e il 73.22% delle pagine mobili hanno adottato HTTPS.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTeddX0_5hUvxpYkd-927hEBlIqzuQiFn5450F2gNc9RJ5K8Wy0ln3KKD-gPWAxJ5Lo3H_km4ljHQEt/pubchart?oid=422955435&format=interactive",
+  sheets_gid="337739550",
+  sql_file="pages_wpt_bodies_by_device.sql"
+) }}
+
+### AMP
+
+[AMP](https://amp.dev/) (precedentemente chiamato Accelerated Mobile Pages) è un framework HTML open source che è stato lanciato da Google nel 2015 per aiutare le pagine a caricarsi più rapidamente, soprattutto sui dispositivi mobile. AMP può essere implementato come una versione alternativa di pagine web esistenti o sviluppati da zero utilizzando il framework AMP.
+
+Quando è disponibile una versione AMP per una pagina, verrà mostrata da Google nei risultati di ricerca per dispositivi mobile, insieme al logo AMP.
+
+È anche importante notare che mentre l'utilizzo di AMP non è un fattore di ranking per Google (o per qualsiasi altro motore di ricerca), la velocità web è un fattore di ranking.
+
+Inoltre, al momento della stesura di questo documento, AMP è un requisito per essere incluso nel carosello di Google Top Stories nei risultati di ricerca per dispositivi mobile, che è una caratteristica importante per le pubblicazioni relative alle notizie. Tuttavia, questo cambierà nel maggio 2021, quando i contenuti non AMP diventeranno idonei a condizione che soddisfino le [norme sui contenuti di Google News](https://support.google.com/news/publisher-center/answer/6204050) e fornisce un'ottima [page experience](https://developers.google.com/search/docs/guides/page-experience) come [annunciato da Google a novembre di quest'anno](https://developers.google.com/search/blog/2020/11/timing-for-page-experience).
+
+Durante il controllo dell'utilizzo di AMP come versione alternativa di una pagina non basata su AMP, abbiamo scoperto che lo 0.69% delle pagine web mobile e lo 0.81% di quelle desktop includevano un tag `amphtml` che puntava a una versione AMP. Sebbene l'adozione sia ancora molto bassa, si tratta di un leggero miglioramento rispetto [ai risultati dello scorso anno](../2019/seo#amp), in cui solo lo 0.62% delle pagine mobile conteneva un link a una versione AMP.
+
+D'altra parte, nel valutare l'utilizzo di AMP come framework per lo sviluppo di siti Web, abbiamo riscontrato che solo lo 0.18% delle pagine mobile e lo 0.07% di quelle desktop specificava `<html amp>` o l'attributo emoji `<html ⚡>`, il quale vengono utilizzati per indicare le pagine basate su AMP.
+
+### Applicazioni Single-page
```

Comment:
@chefleo I wonder if in Italian say Applicatione Single-page or we should keep it in english as Single Page Application. @giopunt thoughts? We might want even ask Twitter for this 😆